versions.lock

```diff
@@ -5,37 +5,45 @@ com.fasterxml.jackson.core:jackson-databind:2.11.3 (3 constraints: 4a3a2999)
 com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.11.3 (1 constraints: cf0c0316)
 com.fasterxml.jackson.module:jackson-module-afterburner:2.11.3 (1 constraints: cf0c0316)
 com.github.ben-manes.caffeine:caffeine:2.9.0 (1 constraints: 0d051036)
+com.google.auto:auto-common:1.0 (2 constraints: 8b1626fd)
 com.google.auto.service:auto-service-annotations:1.0 (1 constraints: a5041a2c)
 com.google.code.findbugs:jsr305:3.0.2 (4 constraints: 5c3586ed)
-com.google.errorprone:error_prone_annotations:2.6.0 (5 constraints: c33cf397)
+com.google.errorprone:error_prone_annotations:2.6.0 (6 constraints: 20472e87)
 com.google.guava:failureaccess:1.0.1 (1 constraints: 140ae1b4)
-com.google.guava:guava:30.1.1-jre (2 constraints: 8814c38e)
+com.google.guava:guava:30.1.1-jre (5 constraints: cb427e81)
 com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava (1 constraints: bd17c918)
 com.google.j2objc:j2objc-annotations:1.3 (1 constraints: b809eda0)
 com.palantir.jvm.diagnostics:jvm-diagnostics:0.0.2 (1 constraints: 0405ed35)
 com.palantir.safe-logging:preconditions:1.14.0 (2 constraints: da117b05)
 com.palantir.safe-logging:safe-logging:1.14.0 (3 constraints: 0b23aa52)
 com.palantir.tracing:tracing:4.22.0 (1 constraints: 3a05453b)
 com.palantir.tracing:tracing-api:4.22.0 (2 constraints: 0912df17)
+com.squareup:javapoet:1.13.0 (1 constraints: 3705323b)
```

Comment:
Is this really just a compile time dependency for the annotation processor?

Reply:
That's correct, it's considered a 'main' dependency because the annotation processor (main sources) depend on it, but it won't be pulled into main dependencies of projects which use the annotation processor.